acs-core/src/main/java/com/arrow/acs/client/model/AuditLogModel.java

```diff
@@ -16,6 +16,7 @@
 public class AuditLogModel extends TsModelAbstract<AuditLogModel> {
 	private static final long serialVersionUID = -6713413123784260777L;
```

Comment:
In general, it is desired to re-generate serialVersionUID if change class fields